packages/safe-ethers-adapters/src/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { SafeTransactionData } from "@gnosis.pm/safe-core-sdk/dist/src/utils/transactions/SafeTransaction"
+import { getCreateCallDeployment } from "@gnosis.pm/safe-deployments"
+import { providers, utils } from "ethers"
+
+export const createLibDeployment = getCreateCallDeployment()
+export const createLibAddress = createLibDeployment!!.defaultAddress
+export const createLibInterface = new utils.Interface(createLibDeployment!!.abi)
+
+const mapStatus = (receipt: providers.TransactionReceipt): number => {
+    const success = receipt.logs.find((log: any) => log.topics[0] === "0x442e715f626346e8c54381002da614f62bee8d27386535b2521ec8540898556e")
```

Comment:
Could you specify in a comment the operation that corresponds to `0x442e715f626346e8c54381002da614f62bee8d27386535b2521ec8540898556e`?

Reply:
I guess it is `ExecutionSuccess(bytes32 txHash, uint256 payment)`